src/control/directive.spec.ts

```diff
@@ -263,6 +263,27 @@ describe(NgrxFormControlDirective.name, () => {
     });
   });
 
+  describe('ngrxUpdateOn "never"', () => {
+    beforeEach(() => {
+      directive.ngOnInit();
+      directive.ngrxFormControlState = { ...INITIAL_STATE, isTouched: true, isUntouched: false };
```

Comment:
This is not necessary here since it leads to your additional check for the update on type not being executed. Just remove this line. The test should still work without it.